src/api/projects/queries/getProject/index.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import Project from '@models/Project'
+import { AUTHENTICATION_ERROR_NO_USER } from '@constants/errors'
+
+type Context = {
+  request: express$Request
+}
+
+type GetProjectArgs = {
+  projectId: string
+}
+
+const resolver = async (
+  _: any,
+  { projectId }: GetProjectArgs,
+  { request }: Context
+) => {
+  if (request.user === null) {
+    throw new Error(AUTHENTICATION_ERROR_NO_USER)
+  }
+
+  const project = await Project.query()
+    .eager('locales')
+    .findOne({ id: projectId, ownerId: request.user.id })
+
+  if (!project) {
+    throw new Error('This project was not found')
```

Comment:
Not sure this is the best way to return an error, wdyt ?

Reply:
I think it’s fine. Users shouldn’t query projects that don’t exist, so we don’t need to show them a clean error client-side IMO.

That would cause a client-side query to fail, though. But that‘s fine, again it should never happen. :)